src/NuGet.Core/NuGet.Common/Errors/PackagingLogMessage.cs

```diff
@@ -33,6 +36,21 @@ private PackagingLogMessage(LogLevel logLevel, NuGetLogCode logCode, string mess
             Time = DateTimeOffset.UtcNow;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the PackLogMessage class
+        /// </summary>
+        /// <param name="logLevel">The log level</param>
+        /// <param name="logCode">The NuGet log code</param>
+        /// <param name="message">The log message</param>
+        /// <param name="libraryId">The package Id</param>
+        /// <param name="framework">The NuGet framework</param>
+        private PackagingLogMessage(LogLevel logLevel, NuGetLogCode logCode, string message, string libraryId, NuGetFramework framework)
+            : this(logLevel, logCode, message)
+        {
+            LibraryId = libraryId;
+            Framework = framework;
```

Comment:
I think it's better to have only constructor that sets fields/properties, and all other constructors call the one with the most parameters, since that way it's easiest to verify that all fields/properties are set. It also makes it more explicit about which properties are allowed to be null (although there's no null checking in this file)

Reply:
Sure, please check now.